spec/base_spec.rb

```diff
@@ -113,6 +117,17 @@
       expect(Log.where(:title => "test tag log").count).to eq 1
       expect(Log.where(:title => "test tag log").first._type).to eq "TagLog"
     end
+
+    context "when accepts_nested_attributes_for" do
+      before { User.accepts_nested_attributes_for :posts }
+
+      it "set association" do
+        user = User.new(:email => "t@1.com", :posts_attributes => { "0" => {:title => "This is title!"}})
```

Comment:
Line is too long. [105/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside } missing.<br>Space inside { missing.